README.rst

```diff
@@ -34,6 +34,17 @@ Using SSH key for authentication
 
     h.executor(user).run_cmd(['echo', 'Use pkey for auth instead of password'])
 
+Using SSH key with disabled algorithms on paramiko SSHClient connect (Used when connecting to machines using old SSH)
+
+.. code:: python
+
+    from rrmngmnt import Host, UserWithPKey
+
+    h = Host("10.11.12.13")
+    user = UserWithPKey('user', '/path/to/pkey'))
+
+    h.executor(user).run_cmd(['echo', 'use of disabled algorithms'], disabled_keys = {'pubkeys':['rsa-sha2-512','rsa-sha2-256']})
```

Comment:
Do we need to run it on all command that we run?
It should be set when the host created.

Reply:
I think putting this on the Executor object is more relavent since all the auth information is saved on it as well

Reply:
Changed location to Host